plugins/by-name/codeium-vim/default.nix

```diff
@@ -58,6 +58,7 @@ lib.nixvim.plugins.mkVimPlugin {
     bin = mkOption {
       type = with types; nullOr str;
       default = "${pkgs.codeium}/bin/codeium_language_server";
+      defaultText = lib.literalExpression ''pkgs.codeium + "/bin/codeium_language_server"'';
```

Comment:
```suggestion
      defaultText = lib.literalExpression ''lib.getExe' pkgs.codeium "codeium_language_server"'';
```

Reply:
Agree.
We should maybe also modify the default to that? But it's a bit orthogonal to this